source-code/sdk-js/src/adapter-sveltekit/utils/shared/utils.ts

```diff
@@ -0,0 +1,23 @@
+import type { RelativeUrl } from '../../../core/index.js'
+import type { SvelteKitClientRuntime } from '../client/runtime.js'
+
+export const inlangSymbol = Symbol.for("inlang")
+
+export const setRuntimeToData = <Data extends Record<string, unknown> | null>(data: Data, runtime: SvelteKitClientRuntime): Data & { [inlangSymbol]: SvelteKitClientRuntime } =>
```

Comment:
how about "setRuntimeInData"?

Reply:
Or better addRuntimeToData

Reply:
I was also thinking of renaming that function